README.template.md

```diff
@@ -4,8 +4,8 @@
 
 ### Requirements
 
-- A Momento Auth Token is required, you can generate one using
-  the [Momento CLI](https://github.com/momentohq/momento-cli)
+- A Momento API Key is required, you can generate one using
+  the [Momento Console](https://console.gomomento.com)
```

Comment:
Can you please also mention here that they need to add the token in their `.env` file like so: `MOMENTO_API_KEY=[generated-token]`?

Reply:
I added [here](https://github.com/momentohq/laravel-cache/pull/28/files#diff-661db4cb1e4c4b70c019f9572d5dadf8fdf408ab60952ac51eb2bddd196cdde4R45-R46) following how we mentioned it in the `laravel-example` repo.